<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogHeaderWidget</class>
 <widget class="QWidget" name="dialogHeaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QCheckBox" name="permanentCheckBox">
         <property name="text">
          <string>Permanent</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="importantCheckBox">
         <property name="text">
          <string>Important</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="identifierLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="conditionLabel">
         <property name="text">
          <string>Condition Script</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="informationLineEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="descriptionLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="informationLabel">
         <property name="text">
          <string>Information Script</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="conditionLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="participantsLabel">
         <property name="text">
          <string>Participants</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="numberLabel">
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="numberSpinBox">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="identifierLabel">
         <property name="text">
          <string>Identifier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="participantsLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveChangesButton">
         <property name="text">
          <string>Save changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveNewDialogButton">
         <property name="text">
          <string>Save new dialog</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveChangesButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogHeaderWidget</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveNewDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogHeaderWidget</receiver>
   <slot>saveNewDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveChanges()</slot>
  <slot>saveNewDialog()</slot>
 </slots>
</ui>
